src/components/datePicker/DatePickerSelectors.ts

```diff
@@ -0,0 +1,22 @@
+import * as _ from 'underscore';
+import {IReactVaporState} from '../../ReactVapor';
+import {IDatePickerState} from './DatePickerReducers';
+
+const getDatePicker = (state: IReactVaporState, props: {id: string}): IDatePickerState => {
+    const datePickers: IDatePickerState[] = _.map(state.datePickers || [], (datePicker: IDatePickerState) => {
+        if (datePicker.id.indexOf(props.id) === 0) {
+            return datePicker;
+        }
+    }).filter(Boolean);
+    return datePickers && datePickers[0] || null;
+};
+
+const getDatePickerLimits = (state: IReactVaporState, props: {id: string}): [Date, Date?] => {
+    const picker = DatePickerSelectors.getDatePicker(state, props);
+    return picker ? [picker.appliedLowerLimit, picker.appliedUpperLimit] : [null, null];

```

Comment:
just to be sure, you have a interface with a required Date as first param but you return null with your selector
Do you think its will be better to return somthing like [0, null] juste to avoid a Date at null ?